src/main/java/com/liferay/ide/idea/ui/modules/LiferayProjectTypeStep.java

```diff
@@ -604,7 +604,7 @@ private boolean _showCustomOptions(@NotNull ModuleBuilder builder) {
 					return false;
 				}
 
-				if (workspaceProvider.isGradleWorkspace()) {
+				if (workspaceProvider.isGradleWorkspace() && workspaceProvider.isFlexibleLiferayWorkspace()) {
```

Comment:
@DL-EthanSun  @simonjhy  what happens if user does not have a flexible workspace (older workspace version)?

Reply:
if the workspace plugiun version is older than 2.5.0, then we don't need to popup workspace product selection dialog.